spec/features/requests/request_spec.rb

```diff
@@ -1400,9 +1287,11 @@
 
         describe 'Request a temp holding item from Resource Sharing - RES_SHARE$IN_RS_REQ' do
           before do
-            borrow_direct = ::BorrowDirect::RequestItem.new("22101008199999")
-            expect(::BorrowDirect::RequestItem).to receive(:new).with("22101008199999").and_return(borrow_direct)
-            expect(borrow_direct).to receive(:make_request).with("Firestone Library", isbn: '9781472523242').and_return('123456')
+            stub_illiad_patron
+            stub_request(:post, transaction_url)
+              .to_return(status: 200, body: responses[:transaction_created], headers: {})
+            stub_request(:post, transaction_note_url)
+              .to_return(status: 200, body: responses[:note_created], headers: {})
```

Comment:
These requests are handled by ILLiad now, so let's mock ILLiad in our test setup.